napari/_vispy/vispy_image_layer.py

```diff
@@ -143,6 +153,22 @@ def _on_attenuation_change(self, event=None):
         if isinstance(self.node, VolumeNode):
             self.node.attenuation = self.layer.attenuation
 
+    def _on_mode_change(self, event=None):
+        if isinstance(self.node, VolumeNode):
+            self.node.mode = self.layer.mode
```

Comment:
Is `mode` an attribute that you're adding to both vispy and napari? Maybe we can give it a more descriptive name `mode` is too generic - I know I often like my single word names, but we might need a two word one.

Also right now `mode` must be one of `{'volume', 'plane'}`. Do you think we'll ever extend that? If not maybe we just need a bool like `planar=False`?

Reply:
yes, for now I have called it `mode` in both the `VolumeVisual` and on the layer. I definitely agree it's not great but have struggled to come up with something which is both descriptive and doesn't require too much understanding... open to any suggestions!

I've got a couple of ideas cooking but I'm not sure where they will go - `spotlight` for highlighting a local region of a volume/plane. I'm not a huge fan of `planar` - what about `render_as_plane`? This could live either on the layer directly or on the `Plane3D` object